crates/astria-sequencer/src/config.rs

```diff
@@ -5,15 +5,20 @@ use serde::{
 };
 
 const DEFAULT_ABCI_LISTEN_ADDR: &str = "127.0.0.1:26658";
+const DEFAULT_GENESIS_ACCOUNT_ALLOCATIONS_PATH: &str = "accounts-genesis.csv";
+const DEFAULT_COMETBFT_GENESIS_FILE: &str = "/.cometbft/config/genesis.json";
 
 #[derive(Debug, Deserialize, Parser, Serialize)]
 pub struct Config {
     /// The endpoint on which Sequencer will listen for ABCI requests
     #[arg(long, default_value_t = String::from(DEFAULT_ABCI_LISTEN_ADDR))]
     pub(crate) listen_addr: String,
     /// The path to the json encoded genesis file with a list of accounts.
-    #[arg(long)]
+    #[arg(long, default_value_t = String::from(DEFAULT_GENESIS_ACCOUNT_ALLOCATIONS_PATH))]
     pub(crate) genesis_file: String,
+    /// The path to the json encoded genesis file with a list of accounts.
+    #[arg(long, default_value_t = String::from(DEFAULT_COMETBFT_GENESIS_FILE))]
+    pub(crate) cometbft_genesis_file: String,
```

Comment:
The naming is confusing - what's the difference between `cometbft_genesis_file` and `genesis_file`?

Reply:
yeah, maybe I'd call the file with the initial accounts `initial_app_state` or something